touchevents.html

```diff
@@ -130,70 +130,43 @@
   </head>
   <body>
     <section id='sotd'>
-     By publishing this Recommendation, W3C expects that the functionality
-     specified in this Touch Interface Recommendation will not be affected by
-     changes to HTML5 or Web IDL as those specifications proceed to
-     Recommendation.
-
-     The WG has completed and approved this specification's 
-     <a href="http://w3c-test.org/webevents/tests/touch-events-v1/approved/">Test Suite</a> 
-     and created an 
-     <a href="http://www.w3.org/2010/webevents/wiki/TEv1ImplReport">
-     Implementation Report</a> that shows that two or more independent implementations 
-     pass each test.
+      <p>
+        By publishing this Recommendation, W3C expects that the functionality specified in this Touch Interface Recommendation will not be affected by changes to HTML5 or Web IDL as those specifications proceed to Recommendation. The WG has completed and approved this specification's <a href="http://w3c-test.org/webevents/tests/touch-events-v1/approved/">Test Suite</a> and created an <a href="http://www.w3.org/2010/webevents/wiki/TEv1ImplReport"> Implementation Report</a> that shows that two or more independent implementations  pass each test.
```

Comment:
What's the benefit of removing the line wrapping here?  It seems to just make reviewing diffs more difficult.  Sorry if I missed / forgot some discussion on this.